api/src/components/publication/service.ts

```diff
@@ -4,7 +4,7 @@ import * as I from 'interface';
 import * as client from 'lib/client';
 import * as referenceService from 'reference/service';
 import * as Helpers from 'lib/helpers';
-import { Links } from '@prisma/client';
+import { Links, Prisma } from '@prisma/client';
```

Comment:
If we import Prisma here, for consistency we should probably not also import Links. We can just import Prisma and when Links is referenced in the file, we write Prisma.Links.

Reply:
Links type does not exist on the Prisma namespace. Only if we import * as Prisma then Prisma.Links will work, but Prisma.PublicationWhereInput will become Prisma.Prisma.PublicationWhereInput which is not ideal either.

Reply:
OK, no problem then!